src/parser/types.ts

```diff
@@ -1,12 +1,13 @@
-export enum FilterType {
-  RegexNotEquals = '!~',
-  RegexEquals = '=~',
-  NotEquals = '!=',
-  Equals = '=',
-}
+export const FilterType = {
+  RegexNotEquals: '!~' as FilterType,
+  RegexEquals: '=~' as FilterType,
+  NotEquals: '!=' as FilterType,
+  Equals: '=' as FilterType,
+};
+export type FilterType = '!~' | '=~' | '!=' | '=';
 export interface QueryFilter {
   filter: FilterType;
-  value: string;
+  value: string | number;
```

Comment:
I thought that filter value could be only `string` that will be passed to the regexp. Why `number` here?

Reply:
good point, fixed that now. changed types slightly as well